### PR TITLE
Make Source translatable

### PR DESCRIPTION
**Pull request by chibiribiri**
10 years and finally u go for a translation of the plattform. A little sad about the fact, that even the newest standard theme doesn't come with the using of the Translate-Helper. So I thought I should contribute.

### CODE REVIEW

**Comment by vinzent** [comment]
uhhhh.. would i have known 1h before.

File: locales/de.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Read more": "Lies mehr",
+  "Featured": "Hervorgehoben",
+  "Powered by {ghostlink}": "Läuft mit {ghostlink}",
+  "Search posts, tags and authors": "Suche nach Beiträgen, Tags und Autoren",
+  "Subscribe": "Abonnieren",
+  "Sign in": "Anmelden",
+  "Account": "Account",
```
Comment (vinzent) [inline locales/de.json:8]: Wird hier mit "Konto" übersetzt: https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/locales/de/portal.json#L13

**Review by vinzent — COMMENTED** [review]
in the partials/lightbox.hbs there are still some strings missing.

File: locales/de.json (hunk 1 above)
Comment (vinzent) [reply locales/de.json:8]: and "Sign in" is translated "Einloggen" here: https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/locales/de/portal.json#L105 . might be good to use same translations.

**Comment by vinzent** [comment]
Hi @chibiribiri , I asked the support if they go to support translations. They answered they won't currently support translations. But they are planning to some time in the future. But no ETA yet.  Would be nice if they would give feedback to contributors here too.

Unfortunately this means to rebase on every upgrade of the Source theme until at some given point in the future they will decide to go for translations.

**Comment by minimaluminium** [comment]
Thanks for your contribution! As mentioned in #31, we're not ready to include translations by default in the theme at the moment.

**Comment by vinzent** [comment]
@minimaluminium would a PR with only the translator helpers be accepted? it would ease the recurring rebase process alot.

**Comment by minimaluminium** [comment]
An intention behind this is keeping the source code of the official themes as simple as possible as they're usually used as a starting point for those who're getting started with Ghost theme development. For this reason, the translation helpers are the main ones we're trying leave as optional for those who want to implement on their own. 

> it would ease the recurring rebase process alot

Official themes don't usually get major updates once they're released. So I hope the rebasing process shouldn't be much complicated to deal with.

**Comment by wblondel** [comment]
Hello @minimaluminium,

I understand official themes don't usually get major updates once they are released, but this theme seems to be outdated since the day it was released (on Oct 3, 2023 according to GitHub). I am wondering why Gulp, abandoned since 2019, was used, as well as [PhotoSwipe](https://github.com/dimsemenov/PhotoSwipe) v4.1.3, [released on January 8, 2019](https://github.com/dimsemenov/PhotoSwipe/releases/tag/v4.1.3) (first release of branch v5.3 was released on Mar 28, 2022).

I forked the repo with updating everything in mind, but I would like to know what led to these choices.

**Comment by vinzent** [comment]
@chibiribiri I started i18n repo for the source theme: https://github.com/SourceTheme-i18n/Source

**Pull request closed without merging**